docs/src/doc/getting-started/setting-up.md

```diff
@@ -6,6 +6,17 @@ or [manually](#setting-up-manually).
 
 The recommended solution is using the custom IntelliJ IDEA plugin.
 
+## Setting-up using the Godot editor.
+
+You can simply create a regular new Godot project. 
+Once done, you can go to `Project/Tools/Kotlin/JVM/Generate JVM Project`.
+![Godot menu](../assets/img/editor-plugin/generation_menu.png)
+
+The following choice will appear:
+![Project dialog](../assets/img/editor-plugin/generation_choice.png)
+
+After this action, all required files should be generated, and you can safely build the project using the `Build` button at the top-right of the editor window.
+
```

Comment:
I'd add that section after the Setting up using Intellij IDEA project wizard or befor the line above.
As the line above suggests the next section rather than this one IMO

Reply:
I'll move it before, I think it's more natural to start with what is directly withing Godot.